iOS/MasterFeed/MasterFeedViewController.swift

```diff
@@ -48,7 +51,7 @@ class MasterFeedViewController: ProgressTableViewController, UndoableCommandRunn
 	}
 
 	override func viewWillAppear(_ animated: Bool) {
-		clearsSelectionOnViewWillAppear = splitViewController!.isCollapsed
+		clearsSelectionOnViewWillAppear = true
```

Comment:
On iPad, the master feed cell is always left selected, because of how the master feed takes you to the timeline in the same view controller.